observability/logging/logging-6.0/log6x-otel-data-model.adoc

```diff
@@ -0,0 +1,374 @@
+
+== Forwarding and Ingestion Protocol
+
+Red Hat OpenShift Logging collects and forwards logs to OpenTelemetry endpoints using link:https://opentelemetry.io/docs/specs/otlp/[OTLP]. OTLP encodes, transports, and delivers telemetry data. This product can also deploy Loki storage, which provides an OTLP endpont to ingest log streams. This document defines the semantic conventions for the logs collected from various OpenShift cluster sources.
+
+[NOTE]
+====
+Logs are forwarded using OTLP over HTTP, as defined by the OpenTelemetry Observability Framework, with Protobuf payloads encoded in JSON format.
+====
+
+== Semantic Conventions
+
+The log collector in this solution gathers the following log streams:
+
+* Container logs
+* Cluster node journal logs
+* Cluster node auditd logs
+* Kubernetes and OpenShift API server logs
+* OpenShift Virtual Network (OVN) logs
+
+These streams are forwarded according to the sementatic conventions defined by OpenTelemetry semantic attributes. The semantic conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry, identified by *Attributes*. For example, a process running in a container includes attributes such as container_name, cluster_id, pod_name, namespace, and possibly deployment or app_name. These *Attributes* are grouped in the *Resource* object, which helps reduce repetition and optimizes log transmission as telemetry data.
+
+The following sections define the attributes that are generally forwarded.
+
+=== Log Entry Structure
+
+All log streams include the following link:https://opentelemetry.io/docs/specs/otel/logs/data-model/#log-and-event-record-definition[log data] fields:
+
+The *Applicable Sources* column indicates which log sources each field applies to:
+
+* `all`: This field is present in all logs.
+* `container`: This field is present in Kubernetes container logs, both application and infrastructure.
+* `audit`: This field is present in Kubernetes, OpenShift API, and OVN logs.
+* `auditd`: This field is present in node auditd logs.
+* `journal`: This field is present in node journal logs.
+
+[cols="1,1,1", options="header"]
+|===
+|Name |Applicable Sources |Comment
+
+|`body`
+|all
+|
+
+|`observedTimeUnixNano`
+|all
+|
+
+|`timeUnixNano`
+|all
+|
+
+|`severityText`
+|container, journal
+|
+
+|`attributes`
+|all
+|(Optional) Present when forwarding stream specific attributes
+|===
+
+=== Attributes
+
+Log entries include a set of resource, scope, and log attributes based on their source, as described in the following table.
+
+The *Location* column specifies the type of attribute:
+
+* `resource`: Indicates a resource attribute
+* `scope`: Indicates a scope attribute
+* `log`: Indicates a log attribute
+
+The *Storage* column indicates whether the attribute is stored in a LokiStack using the default `openshift-logging` tenancy mode and specifies where the attribute is stored:
+
+* `stream label`
+* `structured metadata`
```

Comment:
I feel it would make sense to explain what differs the two. 
- Steam labels: Allow queries be more efficient when we filter by that label
- Structure metadata: Allow queries to filter by that label

Furthermore I feel it would also make sense to expand on the impact that this has to querying. Users are used to having to `| json` in order to filter their results. With OTLP they can `| my_label`  where `my_label` has to be a structure stream or metadata label.

Reply:
Updated as:

* `stream label`: Enables efficient filtering and querying based on specific labels.
* `structured metadata`: Allows for more detailed filtering and storage of key-value pairs, enabling users to use a direct label for streamlined queries without JSON parsing.

With OTLP, users can filter queries directly by labels rather than using JSON parsing, improving the speed and efficiency of queries.